easybuild/easyconfigs/a/Austin/Austin-3.2.0.eb

```
@@ -0,0 +1,31 @@
+easyblock = 'CmdCp'
+
+name = 'Austin'
+version = '3.2.0'
+
+homepage = 'https://github.com/P403n1x87/austin'
+description = "Austin is a Python frame stack sampler for CPython written in pure C."
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/P403n1x87/austin/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['ee189af905fac77ff9173b57d8e927bdb7c4cf5e18b1bfd7f4456ac46fe04484']
+
+cmds_map = [('.*', "$CC -std=c99 -O3 -Os -Wall -pthread src/*.c -o src/austin")]
```

Is it necessary to compile with `-Os`? I would use `$CFLAGS` or `$OPTFLAGS` here.
```suggestion
cmds_map = [('.*', "$CC $CFLAGS -std=c99 -Wall -pthread src/*.c -o src/austin")]
```